include/oneapi/dpl/pstl/utils.h

```diff
@@ -584,6 +584,29 @@ template <typename _Tp>
 using __is_const_callable_object =
     ::std::integral_constant<bool, __is_callable_object<_Tp>::value && __is_pointer_to_const_member<_Tp>::value>;
 
+struct __next_to_last
+{
+    template <typename _Iterator>
+    typename ::std::enable_if<::std::is_same<typename ::std::iterator_traits<_Iterator>::iterator_category,
+                                             ::std::random_access_iterator_tag>::value,
```

Comment:
I think you can use `__is_random_access_iterator` from https://github.com/oneapi-src/oneDPL/blob/main/include/oneapi/dpl/pstl/iterator_defs.h#L69

Reply:
can but I would not to do it... due to the follow objections:
1) it "pulls"  iteratrors_defs.h file.
2) The tests and the test utils should include "internal stuff" -  __is_random_access_iterator , I believe.. 
3) Yes, we may define similar "utility" in the test util... But, there are many places (in the file utils.h) where ::std::enable_if is used....  So, we have to modify all of them.

Reply:
ok. I see